README.md

```diff
@@ -50,6 +50,7 @@ The examples folder contains example solutions across a variety of Google Cloud
 * [QAOA](examples/qaoa) - Examples of parsing a max-SAT problem in a proprietary format.
 * [Redis Cluster on GKE Example](examples/redis-cluster-gke) - Deploying Redis cluster on GKE.
 * [Spinnaker](examples/spinnaker) - Example pipelines for a Canary / Production deployment process.
+* [TensorFlow unit testing](examples/tensorflow-profiling-examples) - How to write unit tests for TensorFlow models.
```

Comment:
I think this should link to `tensorflow-unit-testing`
Also, it looks like the convention is to capitalize all words in the name (i.e. TensorFlow Unit Testing)

Reply:
fixed